firebase-database/firebase-database.gradle.kts

```diff
@@ -50,18 +50,14 @@ android {
 }
 
 dependencies {
-  implementation("com.google.firebase:firebase-common:20.3.1")
+  implementation("com.google.firebase:firebase-common:20.3.2")
   implementation("com.google.firebase:firebase-components:17.1.0")
+  implementation("com.google.firebase:firebase-auth-interop:20.0.0")
   implementation("com.google.firebase:firebase-appcheck-interop:16.1.1")
   implementation("com.google.firebase:firebase-database-collection:18.0.1")
 
-  implementation("androidx.annotation:annotation:1.1.0")
-  implementation("com.google.android.gms:play-services-base:18.0.1")
-  implementation("com.google.android.gms:play-services-tasks:18.0.1")
-  implementation("com.google.android.gms:play-services-basement:18.1.0")
```

Comment:
Even though we can get this deps transitively from common, I think is better to leave them here. But using the version catalog

Reply:
They actually are here still, I just moved them to a `bundle`:
```toml
playservices = ["playservices-base", "playservices-basement", "playservices-tasks"]
```